configure.ac

```diff
@@ -4567,6 +4567,18 @@ AC_SUBST([INSTALL_SDL2_CONFIG], [$enable_sdl2_config])
 
 # Verify that we have all the platform specific files we need
 
+if test x$have_audio != xyes; then
+    if test x$enable_audio = xyes; then
+        AC_DEFINE(SDL_AUDIO_DRIVER_DUMMY, 1, [ ])
+    fi
+    SOURCES="$SOURCES $srcdir/src/audio/dummy/*.c"
```

Comment:
CheckDummyVideo() and CheckDummyAudio() already do this ???

Reply:
CheckDummyVideo/Audio adds dummy if they are explicitly enabled (enable-dummyaudio/--enable-video-dummy). Here it is added if the subsystem is enabled.

Reply:
CheckDummyVideo/Audio adds dummy if they are explicitly enabled (enable-dummyaudio/--enable-video-dummy). Here it is added if the subsystem is enabled.